zaza/openstack/utilities/openstack.py

```diff
@@ -2651,6 +2652,50 @@ def upload_image_to_glance(glance, local_path, image_name, disk_format='qcow2',
     return image
 
 
+def is_ceph_image_backend():
+    """Check if glance is related to ceph, therefore using ceph as backend.
+
+    :returns: True if glance is related to Ceph, otherwise False
+    :rtype: bool
+    """
+    status = juju_utils.get_full_juju_status(model.get_juju_model())
+    result = False
+    try:
+        result = 'ceph-mon' in (
+            status['applications']['glance']['relations']['ceph'])
+    except KeyError:
+        pass
+    if result is False:
+        logging.info("Did not detect Ceph related to Glance")
+    return result
+
+
+def _convert_image_if_needed(local_path, disk_format):
```

Comment:
I don't think this function name conveys what the function is doing very well.  It seems like it is more of a `_ensure_image_is_raw_format()` ; I'm not set on *that* name, but it does describe the function more accurately.  Also, why is it nominally private? (i.e. with a leading `_`)?